block/manager.go

```diff
@@ -276,6 +270,7 @@ func (m *Manager) waitForSync(ctx context.Context) error {
 		// the SL still hasn't got any batches for this chain.
 		m.logger.Info("No batches for chain found in SL. Start writing first batch")
 		atomic.StoreUint64(&m.syncTarget, uint64(m.genesis.InitialHeight-1))
+		atomic.StoreInt64(&m.lastSubmissionTime, time.Now().Unix())
```

Comment:
Seems to me like only if we start a fresh sequencer the `lastSubmissionTime` will get initialized, otherwise if we restart a sequencer which already produced blocks, the `lastSubmissionTime` will not be initialized. 

Why not just put it in the beginning of the `produceBlockLoop`?

Reply:
It's also in the `updateSyncParams` so it should cover all non-fresh executions
I'll move it to the `produceBlockLoop` anyway

Reply:
true. but if we get an error (different then `ErrBatchNotFound`) in 'waitForSync' the  `updateSyncParams` will not be called but we'll still continue producing blocks (despite the error). 

don't remember the reason for that (maybe a bug) but this is the current behavior.